dotcom-rendering/cypress/integration/parallel-5/atom.video.spec.js

```diff
@@ -21,6 +22,71 @@ const interceptPlayEvent = (id) => {
 	);
 };
 
+const parseCustParams = (custParamsString) => {
+	const custParams = decodeURIComponent(custParamsString);
```

Comment:
You can use [`URLSearchParams`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) which has a nicer API (and we could remove this function).

Reply:
Thanks for the suggestion! That works nicely for the `cust_params` field.